releasetasks/templates/firefox/enUS.yml.tmpl

```diff
@@ -288,14 +284,10 @@
                     type: directory
                     expires: "{{ never }}"
             env:
-                {% if signing_class == "dep-signing" %}
-                MOZ_DISABLE_MAR_CERT_VERIFICATION: true
-                SIGNING_CERT: "dep"
-                {% elif signing_class == "nightly-signing" %}
-                SIGNING_CERT: "nightly"
-                {% elif signing_class == "release-signing" %}
-                SIGNING_CERT: "release"
+                {% if moz_disable_mar_cert_verification is defined %}
```

Comment:
I'd use `moz_disable_mar_cert_verification is defined and moz_disable_mar_cert_verification` instead, because `None` could be dangerous.